packages/tools/devtools/devtools-view/package.json

```diff
@@ -91,10 +100,15 @@
 		"globby": "^13.2.2",
 		"jest": "^26.6.3",
 		"jest-junit": "^10.0.0",
+		"mocha": "^10.2.0",
+		"mocha-json-output-reporter": "^2.0.1",
+		"mocha-multi-reporters": "^1.5.1",
+		"moment": "^2.21.0",
```

Comment:
Was this one necessary? Peer dep or something?

Reply:
Yeah, it's a peer dep of mocha-multi-reporters